datadog/resource_datadog_logs_custom_pipeline.go

```diff
@@ -186,6 +191,26 @@ var grokParser = &schema.Schema{
 	},
 }
 
+var lookupProcessor = &schema.Schema{
+	Type:     schema.TypeList,
+	MaxItems: 1,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name":       {Type: schema.TypeString, Optional: true},
+			"is_enabled": {Type: schema.TypeBool, Optional: true},
+			"source":     {Type: schema.TypeString, Required: true},
+			"target":     {Type: schema.TypeString, Required: true},
+			"lookup_table": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"default_lookup": {Type: schema.TypeString, Optional: true},
```

Comment:
Should this be marked as required? It looks like trying to create a lookup processor without this value doesn't work.

Reply:
It is optional on the API side, what is the error that you get when you try to create a lookup processor without default lookup?

Reply:
I'm getting this:

```
Error: error updating logs pipeline: (API error 400 Bad Request: {"error":{"code":"InvalidArgument","message":"Invalid Pipeline","details":[{"code":"InvalidArgument","message":"Invalid default lookup: must not be blank"}]}})
```